pkg/operator/tpr.go

```diff
@@ -59,7 +59,7 @@ func (o *Operator) createTPR() error {
 		}
 	}
 
-	return err
```

Comment:
i'm reading this change a different way @kokhang.  it looks to me like the only error that will be returned is if it is not an "already exists" error...which is what we want right?